setup.py

```diff
@@ -22,7 +22,15 @@
         "Programming Language :: Python :: 3 :: Only",
     ],
     include_package_data=True,
-    install_requires=['evfl~=1.1', 'PyYAML~=5.1', 'aamp~=1.0', 'byml~=2.0', 'wszst_yaz0~=1.1'],
+    install_requires=[
+        'evfl~=1.1',
+        'pyqt5-sip~=12.7',
```

Comment:
Is the pyqt5 package required?

Reply:
Yes, but since it was already required when you originally published the package I thought there was a reason you left it out, plus it would be installed to fulfill the other PyQt stuff, I left it out.

Reply:
Ah, that makes sense. I did leave it out to make sure Linux users don't have to download large binary blobs for PyQt5 (which is sometimes already installed at the system level).